typescript/src/processors/readTypeScriptModules/index.spec.ts

```diff
@@ -300,7 +300,7 @@ describe('readTypeScriptModules', () => {
         processor.$process(docs);
 
         const foo: MethodMemberDoc = docs.find(doc => doc.name === 'foo');
-        expect(foo.parameters).toEqual(['num1: number | string', 'num2?: number']);
+        expect(foo.parameters).toEqual(['num1: string | number', 'num2?: number']);
```

Comment:
For some reason the ordering of these changes, but it is not a concrete breaking change, IMO.